src/constants.rs

```diff
@@ -6,7 +6,7 @@ pub const FUELS_VERSION_FILE: &str = "fuels_version";
 
 pub const CHANNEL_LATEST_URL: &str =
     "https://raw.githubusercontent.com/FuelLabs/fuelup/gh-pages/channel-fuel-latest.toml";
```

Comment:
Should we remove this, and the other code for `latest`? It might be confusing to leave it in there.

Reply:
Nice catch!